components/infobox/wikis/valorant/infobox_series_custom.lua

```diff
@@ -0,0 +1,69 @@
+---
+-- @Liquipedia
+-- wiki=valorant
+-- page=Module:Infobox/Series/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Series = require('Module:Infobox/Series')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Chronology = require('Module:Infobox/Widget/Chronology')
+
+local CustomSeries = {}
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomSeries.run(frame)
+	local series = Series(frame)
+	_args = series.args
+	series.createWidgetInjector = CustomSeries.createWidgetInjector
+	series.createBottomContent = CustomSeries.createBottomContent
```

Comment:
unused

Reply:
```suggestion
```

Reply:
Leftover from thinking stage of the Chronology implementation (however I remember then that BottomContent is outside the infobox). Guess I forgot to remove it.